packages/deployments/contracts/contracts/core/connext/helpers/RelayerProxy.sol

```diff
@@ -273,19 +319,39 @@ contract RelayerProxy is ProposedOwnable, ReentrancyGuard, GelatoRelayFeeCollect
   }
 
   /**
-   * @notice Updates the Autonolas contract address on this contract.
-   * @param _autonolas - New Autonolas contract address.
+   * @notice Updates the propose cooldown.
+   * @param _proposeCooldown The new cooldown in seconds.
    */
-  function setAutonolas(address _autonolas) external onlyOwner definedAddress(_autonolas) {
-    _setAutonolas(_autonolas);
+  function setProposeAggregateRootCooldown(uint256 _proposeCooldown) external onlyOwner {
+    _setProposeAggregateRootCooldown(_proposeCooldown);
   }
 
   /**
-   * @notice Updates the Autonolas priority on this contract.
-   * @param _autonolasPriority - New Autonolas priority.
+   * @notice Updates the finalize cooldown.
+   * @param _finalizeCooldown The new cooldown in seconds.
    */
-  function setAutonolasPriority(uint8 _autonolasPriority) external onlyOwner {
-    _setAutonolasPriority(_autonolasPriority);
+  function setFinalizeCooldown(uint256 _finalizeCooldown) external onlyOwner {
+    _setFinalizeCooldown(_finalizeCooldown);
+  }
+
+  /**
+   * @notice Updates the propose cooldown period on this contract.
+   *
+   * @param _proposeAggregateRootCooldown - Delay for propose.
+   */
+  function _setProposeAggregateRootCooldown(uint256 _proposeAggregateRootCooldown) internal {
+    emit ProposeAggregateRootCooldownChanged(_proposeAggregateRootCooldown, proposeAggregateRootCooldown);
+    proposeAggregateRootCooldown = _proposeAggregateRootCooldown;
+  }
+
+  /**
+   * @notice Updates the finalize cooldown period on this contract.
+   *
+   * @param _finalizeCooldown - Delay for finalize.
+   */
+  function _setFinalizeCooldown(uint256 _finalizeCooldown) internal {
+    emit FinalizeCooldownChanged(_finalizeCooldown, finalizeCooldown);
+    finalizeCooldown = _finalizeCooldown;
```

Comment:
should we allow `withdraw` to take in an address so ERC20s can be removed as well?

Reply:
this is not a hard requirement, but we have been bitten by this before